### DISCUSSION TITLE
How to show remaining inventory qty when importing an BL order

### DISCUSSION THREAD

**Discussion by StoneyBricks** [rgriebl/brickstore#430] [Q&A]
How do I add a column showing the “remaining qty” when importing a BL order?

**Comment by chiminirc** [comment]
There is another program that I use that does this really well.  They added 2 columns because we ended up realizing you usually want to know 2 things.
1. How many left after PICKING this order
2. How many left after ALL orders are picked

That way if you have a drawer of 30, and an order of 10.  You'd see
30 ordered, 20 remaining, 10 in stock

That adds the ability to know if you inventory is off or as a reminder that you'll be coming back to that drawer.

**Comment by chiminirc** [comment]
Otherwise, you can load the order.  Import your inventory.  COPY VALUES FROM DOCUMENT, and copy the quantities
That would leave you with QTY ORIG = the amount in the order and QTY = what is left in stock

![image](https://user-images.githubusercontent.com/76887746/137019692-61ec8e70-79b5-4d36-9c31-806dd7bf0506.png)

**Reply by StoneyBricks** [reply]
Thank you. I have tried this, but I found the column headers (QTY Orig and Qty) confusing. I am hoping a “special” order layout could be created. 

In regards to the “other” application, which one is that?
